src/Options/index.js

```diff
@@ -178,6 +179,8 @@ export interface ParseServerOptions {
   passwordPolicy: ?PasswordPolicyOptions;
   /* Adapter module for the cache */
   cacheAdapter: ?Adapter<CacheAdapter>;
+  /* Adapter module for the schema cache */
+  schemaCacheAdapter: ?Adapter<SchemaCacheAdapter>;
```

Comment:
Did you run `npm run definitions`? See [CONTRIBUTING](https://github.com/parse-community/parse-server/blob/alpha/CONTRIBUTING.md#parse-server-configuration).

Reply:
earlier I forgotten about madge and run definitions. Now circulars are resolved. 
Locally I executed tests on Mongo (recent) and Postgres (npm test). Seem that work.